css/styles.css

```diff
@@ -0,0 +1,73 @@
+body {
+  font-family: 'Inter', sans-serif;
+}
+
+header {
+  width: 100%;
+  padding: 1.5em;
+  height: 2em;
+  display: flex;
+  align-items: center;
+}
+
+header h2 {
+  font-weight: 600;
+  color: #28352f;
+  flex-grow: 1;
+}
+
+header a {
+  width: 2em;
+  height: 2em;
+  background-color: transparent;
+  background-image: url(../images/icons/hamburger.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  border: none;
+}
+
+.headline {
+  height: 100vh;
+  padding: 3em;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+  background-image: url(../images/header-illsutration-mobile.svg);
+  background-position: center;
+  background-size: auto 90%;
+  background-repeat: no-repeat;
```

Comment:
[REQUIRED] Please decrease the size of your background so it looks exactly like [Figma template ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468)

![image](https://user-images.githubusercontent.com/74744787/158710908-621a9d26-fe8c-4e61-a156-5438889fe16d.png)


![image](https://user-images.githubusercontent.com/74744787/158710811-f68fdef9-6c80-474d-a933-cadcc0427801.png)